pyirf/gammapy.py

```diff
@@ -108,7 +107,7 @@ def create_energy_dispersion_2d(
     fov_offset_bins,
 ):
     """
-    Create a ``gammapy.irf.EnergyDispersion2D`` from pyirf outputs.
+    Create a :py:class`gammapy.irf.EnergyDispersion2D` from pyirf outputs.
```

Comment:
Apart from the merging conflicts, there is onlny a typo here,

`:py:class`**:**